internal/mongodoc/doc.go

```diff
@@ -200,6 +200,12 @@ type BaseEntity struct {
 	// set of resource names holding the currently published resource
 	// version for that channel and resource name.
 	ChannelResources map[params.Channel][]ResourceRevision
+	
+	// NoIngest is set to true when a charm or bundle has been uploaded
+	// with a POST request. Since the ingester only uses PUT requests
+	// at present, this signifies that someone has taken over control from
+	// the ingester.
+	NoIngest bool	`bson:",omitempty"`
```

Comment:
I'm not sure. I considered Posted or Pushed, but I decided against that because we're actually interested in what will be returned from the can-ingest endpoint, and we might decide to set the NoIngest flag based on other things too (for example, we might let people explicitly turn off ingestion without uploading something, in which case "Posted" would be inaccurate).